### PR TITLE
Update FAQ  README.md

### PR DESCRIPTION
**Pull request by JamesH65**
SD card robustification in FAQ

### CODE REVIEW

File: faqs/README.md [hunk 1]
```diff
@@ -381,6 +382,10 @@ There is a standard 3.5mm jack for audio out to an amplifier (not on Zero models
...
 
+### What about unplanned power interruptions?
+
+Power interruptions can cause problems on all sorts of electronic device, and the Pi is no different. SD card corruption can be caused if the power is simply turned off without going through a normal shutdown. This is because the system may be writing to the SD card at the point of power failure, leaving the SD card in an invalid state. So, if you cannot prevent power interruptions, one way of making the system more robust is to limit the amount of writing that is done to the SD card, or even stopping it altogether. You can disable swap, enabled logging to RAM, and disable systemd-timesyncd, all of which greatly reduce the number of access to the SD card. You can also make the entire Raspbian installation read only, preventing any writing to the card at all. An internet search will provide instructions on how to implemenat these measures.
```
Comment (lurch) [inline faqs/README.md]: enabled logging -> enable logging
number of access -> number of accesses
implemenat -> implement

And a minor technical point - is it the SD card that gets into an invalid state, or the filesystem that gets into an invalid state? Although perhaps the audience this is aimed at isn't aware of the difference between SD card and filesystem? :man_shrugging:

Comment (JamesH65) [reply faqs/README.md]: Fixed the typo's,  added "filesystem" after SD card, although I think given the audience it's not completely necessary.

**Comment by ita1024** [comment]
+1

**Comment by JamesH65** [comment]
Fixes #1155

**Pull request merged**